packages/contracts-bedrock/test/L1/OptimismPortal2.t.sol

```diff
@@ -3,6 +3,7 @@ pragma solidity 0.8.15;
 
```

Comment:
@coderabbitai Are all of the tests added in PR https://github.com/ethereum-optimism/optimism/pull/10143 also included in this file?